http/src/request/channel/reaction/mod.rs

```diff
@@ -10,22 +10,25 @@ pub use self::{
     delete_all_reactions::DeleteAllReactions, delete_reaction::DeleteReaction,
     get_reactions::GetReactions,
 };
-
 use std::fmt::Write;
-use twilight_model::channel::ReactionType;
+use twilight_model::id::EmojiId;
+
+pub enum CreateReactionType {
```

Comment:
could you add some documentation here about what this serves as compared to tho the similar enum in model?